ignite/contrib/handlers/param_scheduler.py

```diff
@@ -402,6 +402,31 @@ def get_param(self):
         return self.start_value + ((self.end_value - self.start_value) / 2) * (1 - math.cos(math.pi * cycle_progress))
 
 
+class _ParamGroupsOptimizer:
+    """Internal tools to concat a list of parameters from different optimizers.
+    """
+
+    def __init__(self, schedulers):
+        self.schedulers = schedulers
+        self.param_groups = [s.optimizer_param_groups[0] for s in schedulers]
+
+    def state_dict(self):
+        return self.param_groups
+
+    def load_state_dict(self, state):
+        self.param_groups = state
+
+
+def _create_optimizer_list(schedulers):
```

Comment:
@sdesrozis i'm not sure to understand the purpose of this method as it is used only inside `ConcatScheduler` where we need a single optimizer only.

Reply:
You are right, this method should be in class `ConcatScheduler`.